src/tasks/pair-liquidity-info-history-validator.service.ts

```diff
@@ -24,13 +26,13 @@ export class PairLiquidityInfoHistoryValidatorService {
   @Cron(EVERY_5_MINUTES_STARTING_AT_02_30)
   async runTask() {
     try {
-      if (!this.isValidationRunning) {
-        this.isValidationRunning = true;
+      if (!this.tasksService.isRunning) {
+        this.tasksService.startRunning();
```

Comment:
Wording seems a bit weird. Maybe setIsRunning or startedRunning or reserveLock & releaseLock would be better.